codec-framework/pom.xml

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>quilt-parent</artifactId>
+        <groupId>org.interledger</groupId>
+        <version>0.1.0-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <name>Hyperledger Quilt Codec Framework</name>
```

Comment:
Not a blocker, but we should decide if we're using 2 spaces or 4 spaces for tabs in XML and Java. Some of the poms do it one way, and some of the poms do it the other way. I don't have a strong opinion, other than consistency.